modules/onboarding/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "is_organizational" {
+  type        = bool
+  default     = false
+  description = "(Optional) True/False whether secure-for-cloud should be deployed in an organizational setup"
+}
+
+variable "region" {
+  type        = string
+  default     = "us-sanjose-1"
+  description = "(Optional) Default region for resource creation"
```

Comment:
same here, do we need `region` as input to the module?

Reply:
Actually we don't - I think we can get rid off this. I added initially as part of the boilerplate but this can be removed

thanks!

Reply:
great, thanks!